Food_Tracker_App/app/src/main/res/layout/ingredient.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical">
+
+    <TextView
+        android:id="@+id/ingredientDescription"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:paddingTop="20dp"
+        android:paddingLeft="10dp"
+        android:text="Description"
+        android:textSize="20dp" />
+
+    <RelativeLayout
+        android:id="@+id/expand_ingredients"
```

Comment:
we should decide if we want to do either `camelCase` or `snake_case` for ids, try not to use both if possible